base/commands/migration/start_stages_it_test.go

```diff
@@ -42,7 +43,7 @@ func TestMigrationStages(t *testing.T) {
 				"testdata/start/migration_success_initial.json",
 				"testdata/start/migration_success_failure.json",
 			},
-			expectedErr: errors.New("Failed migrating IMAP: imap5: some error"),
+			expectedErr: errors.New("Failed migrating IMAP: imap5: [\"some error\"]"),
```

Comment:
I think this error message is not optimal.
Can't we keep the original string?

Reply:
I would rewrite `fetchMigrationErrors` function like:
```
func fetchMigrationErrors(ctx context.Context, ci *hazelcast.ClientInternal, migrationID string) (string, error) {
	q := fmt.Sprintf(`SELECT JSON_QUERY(this, '$.errors' WITH WRAPPER) FROM %s WHERE __key='%s'`, StatusMapName, migrationID)
	row, err := querySingleRow(ctx, ci, q)
	if err != nil {
		return "", err
	}
	// ex. converts ["some error", "another error"] -> some error\nanother error
	r := strings.Split(
		strings.ReplaceAll(
			strings.ReplaceAll(
				strings.ReplaceAll(string(row.(serialization.JSON)), `"`, ``),
				"[", ""),
			"]", ""),
		",")
	return strings.Join(r, "\n"), nil
}
```

Reply:
If it is possible to iterate `errors` it would be more convenient of course.